appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-document/controller/index.js

```diff
@@ -0,0 +1,37 @@
+import {
+	postDocumentUpload,
+	renderDocumentUpload
+} from '#appeals/appeal-documents/appeal-documents.controller.js';
+
+/** @type {import('@pins/express').RequestHandler<{}>}  */
+const get = async (request, response) => {
+	const { currentComment, currentAppeal, session } = request;
+
+	return renderDocumentUpload({
+		request,
+		response,
+		appealDetails: currentAppeal,
+		backButtonUrl: `/appeals-service/appeal-details/${currentAppeal.appealId}/interested-party-comments/${currentComment.id}`,
+		nextPageUrl: `/appeals-service/appeal-details/${currentAppeal.appealId}/interested-party-comments/${currentComment.id}/add-document/redaction-status`,
+		pageHeadingTextOverride: 'Upload supporting document',
+		allowMultipleFiles: false,
+		documentType: session.costsDocumentType
+	});
+};
+
+/**
+ * @type {import('@pins/express/types/express.js').RequestHandler<{}>}
+ */
+const post = async (request, response) => {
+	const { currentAppeal, currentComment } = request;
+
+	await postDocumentUpload({
+		request,
+		response,
+		nextPageUrl: `/appeals-service/appeal-details/${currentAppeal.appealId}/interested-party-comments/${currentComment.id}/add-document/redaction-status`
+	});
+};
+
+export { get as renderAddDocument, post as postAddDocument };
```

Comment:
It would be simpler for the controller functions to just be named like this, although there might be a reason why it's been done this way